BackfillPowerBI.yml

```diff
@@ -0,0 +1,27 @@
+parameters:
+- name: DeleteAllOrganizationData
+  displayName: Delete all data in Power BI Simeon Data source
+  default: false
+  type: boolean
+
+name: $(Date:yyyyMMdd)$(Rev:.r)
+trigger: none
+pool:
+  vmImage: windows-latest
+resources:
+  repositories:
+  - repository: Templates
+    type: github
+    name: simeoncloud/AzurePipelineTemplates
+    ref: refs/heads/feature/uploadResultsToPowerBI
+    endpoint: simeoncloud
+jobs:
+- job: Job
+  timeoutInMinutes: 60
+  steps:
+  - checkout: Templates
+  - template: Reporting.yml@Templates
```

Comment:
this doesn't seem right...where is Reporting.yml? I'd expect this to call SimeonCloudReportingBackfill

Reply:
yeah, you were right, had missed a push.